From c58ca3d659b03b254a702fded114acfe5f19d16a Mon Sep 17 00:00:00 2001
From: Teo Hall <teo.hall@nxp.com>
Date: Thu, 3 May 2018 14:44:21 -0500
Subject: [PATCH 3708/5242] MLK-18197: Use MU1 for SCFW API calls

commit  bdf2b6adf31edef875d5a2fc02a07371be0cf251 from
https://source.codeaurora.org/external/imx/linux-imx.git

Use correct MU for SCFW API calls to comply with boot
container intended usage.

Signed-off-by: Teo Hall <teo.hall@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi  |    4 ++--
 arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 67fe435..190a6d2 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -103,9 +103,9 @@
 		interrupt-parent = <&gic>;
 	};
 
-	mu: mu@5d1b0000 {
+	mu: mu@5d1c0000 {
 		compatible = "fsl,imx8-mu";
-		reg = <0x0 0x5d1b0000 0x0 0x10000>;
+		reg = <0x0 0x5d1c0000 0x0 0x10000>;
 		interrupts = <GIC_SPI 176 IRQ_TYPE_LEVEL_HIGH>;
 		interrupt-parent = <&wu>;
 		fsl,scu_ap_mu_id = <0>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
index 21b90cc..202327a 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp.dtsi
@@ -114,9 +114,9 @@
 		interrupt-parent = <&gic>;
 	};
 
-	mu: mu@5d1b0000 {
+	mu: mu@5d1c0000 {
 		compatible = "fsl,imx8-mu";
-		reg = <0x0 0x5d1b0000 0x0 0x10000>;
+		reg = <0x0 0x5d1c0000 0x0 0x10000>;
 		interrupts = <GIC_SPI 176 IRQ_TYPE_LEVEL_HIGH>;
 		interrupt-parent = <&wu>;
 		fsl,scu_ap_mu_id = <0>;
-- 
1.7.9.5

